### PR TITLE
Update welcome page

### PR DESCRIPTION
**Pull request by rhcarvalho**
Matching https://github.com/openshift/rails-ex/pull/21

@bparees @ewolinetz @jhadvig PTAL

### CODE REVIEW

File: app/View/Layouts/default.ctp [hunk 1]
```diff
@@ -252,7 +258,7 @@ $ git push</pre>
                   <p>
                   It will also be necessary to update your application to talk to your database back-end.  The <code>config/database.yml</code> file used by rails was set up in such a way that it will accept environment variables for your connection information that you pass to it.
                   Once an administrator has created a MySQL database service for you to connect with you can add the following environment variables to your deploymentConfig to ensure all your frontend pods have access to these environment variables.
-                  
+
 <pre>
 osc env dc/frontend DATABASE_SERVICE_NAME=&lt;database service name&gt;
```
Comment (mfojtik) [inline app/View/Layouts/default.ctp]: s/osc/oc/

**Comment by mfojtik** [comment]
@rhcarvalho LGTM (just one renaming nit)

File: app/View/Layouts/default.ctp (hunk 1 above)
Comment (rhcarvalho) [inline app/View/Layouts/default.ctp]: I'm not touching this in this PR... but I can do a follow up addressing osc -> oc everywhere (there is already other open PRs doing this)

Comment (mfojtik) [inline app/View/Layouts/default.ctp]: ok, I was thinking that while you are there we can save one PR/commit ;-)

**Pull request merged**